internal/atc/tower/airway_view.go

```diff
@@ -53,7 +53,13 @@ func MakeAirwayListView(dim tea.WindowSizeMsg) AirwayListView {
 					return ExecMsg(func(c Commands) tea.Cmd { return c.GetAirwayList })
 				},
 			},
-			Back: nil,
+			Back: &Nav{
```

Comment:
This is actually an interesting approach.

I am not opposed to it in essence. Although I do think the more idiomatic bubbletea approach would be to handle the escape event in the update method and return the same model and the Quit event back up the change.

Essentially mapping escape to quit.

Although using the navigation mechanism was pretty smart.

Let me know what you prefer!

Reply:
The airway list view is the top level views so I would do it in the update method of the AirwayListView.

Ie. the airwaylist view should listen for escape events and transform that into a quit event.

Reply:
Hey @davidmdm, thanks for merging my PR. And sorry for not answering. I was sick.
I also had a version where the escape was part of the airwaylist update method, but i wanted it to be part of the legend (as you have also mentioned below).

Again, Thanks for merging :)